Client/Frontend/Browser/BrowserLocationView.swift

```diff
@@ -132,6 +129,9 @@ class BrowserLocationView : UIView, UIGestureRecognizerDelegate, UITextFieldDele
         editTextField.toolbarTextFieldDelegate = self
         addSubview(editTextField)
 
+        borderLayer = UIView()
```

Comment:
By placing the border layer on top of the edit text field you are basically preventing any touch interactions occuring on the text field. Try pressing the clear text field button, or performing any text editing actions (long press, move cursor, highlight) and you'll notice that they are all disabled.

Instead, take a look at the constraints and how we are setting then in `updateConstraints()`. You can see that when we set the trailing constraint for the text field we set it to be equal to the container, so that they line up. _But_ the container has rounded corners and the text field doesn't, and the border is 1px wide. 

How might that affect the visual display for the border?
How might we be able to use our constraints to fix this (take a look at how the other constraints are being set)?

Not giving you too much info cos I think you can solve this one by yourself.